src/components/ui/NavBar.tsx

```diff
@@ -0,0 +1,71 @@
+import { type ReactNode } from "react";
+import { type IconType } from "react-icons/lib";
+import { TbScubaMask, TbHistory, TbAward } from "react-icons/tb";
+
+type NavItemProps = {
+  title: string;
+  Icon: IconType;
+  pageNo: number;
+  pageHandler: (pageNo: number) => void;
+};
+
+type NavBarProps = {
+  children: ReactNode;
+  page: number;
+  pageHandler: (pageNo: number) => void;
+};
+
+const NavItem = ({ title, Icon, pageNo, pageHandler }: NavItemProps) => {
+  const onChangePage = () => {
+    pageHandler(pageNo);
+  };
+  return (
+    <div className="flex w-1/3 flex-col justify-center" onClick={onChangePage}>
```

Comment:
Is the `w-1/3` necessary? Not a good idea to hardcode the widths like that. Imagine if you need to add a new tab in the Navbar, then you have to go to all the 1/3 and change to 1/4. Using flexbox with the children's with as 100% might solve this